### PR TITLE
Fix remediation file generator by adding proper css chunks

### PR DESCRIPTION
**Pull request by karelhala**
Currently the build creates `1.css` which is incorrect, so I've changed the css generation to create `index.css`.

### CODE REVIEW

File: packages/remediations/.gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 /esm
 /cjs
 /*.css
+*.txt
```
Comment (Hyperkid123) [inline packages/remediations/.gitignore:10]: was this change intended?

Comment (karelhala) [reply packages/remediations/.gitignore:10]: Yup, the build now creates `.txt` files containing some license info from minimize plugin.
